assets/demo-todo-app.spec.js

```diff
@@ -168,13 +168,20 @@ test.describe('Item', () => {
 
     const firstTodo = page.getByTestId('todo-item').nth(0);
     const secondTodo = page.getByTestId('todo-item').nth(1);
-    await firstTodo.getByRole('checkbox').check();
+    const firstTodoCheck = firstTodo.getByRole('checkbox');
+    const secondTodoCheck = secondTodo.getByRole('checkbox');
+
+    await firstTodoCheck.check();
+    await expect(firstTodoCheck).toBeChecked();
```

Comment:
I feel like this assertion does not make a lot of sense, isn't asserting for the completed class already enoughj?

Reply:
ok perhaps. no idea to be honest.